modules/rational-compile.el

```diff
@@ -88,8 +119,8 @@ D could be a single directory or a list of directories."
   "Returns a list of configured directories to autocompile."
   (flatten-list `(,(and rational-compile-modules
                         (expand-file-name "modules/" user-emacs-directory))
-                  ,(and rational-compile-user-configuration
-                        (expand-file-name "./" rational-config-path))
+                  ,(and rational-compile-user-modules
+                        (expand-file-name "modules/" rational-config-path))
```

Comment:
After #136, this should be `custom-modules`. If we prefer just modules, will need an update to `init.el` as well to handle that name change.